File: FrontEnd/src/models/roleDTO.ts
import { Navigation } from "@models/navigationDTO";

export interface RoleDTO {
  navigation: Navigation[];
  role: RoleDto;
}

export interface RoleDto {
  id: string;
  roleName: string | null;
  status: number | null;
  isDeleted: boolean;
  isAdmin: boolean | null;
  numberRole: number | null;
  roleCode: string | null;
}

File: BackEnd/ExamManagement/DPD.HR.Business.Dto/Dto/RoleDto.cs
namespace DPD.HumanResources.Dtos.Dto
{
    public class RoleDto
    {
        public Guid Id { get; set; }
        public string RoleName { get; set; } = string.Empty;
        public string RoleCode { get; set; } = string.Empty;
        public int Status { get; set; }
        public bool IsDeleted { get; set; }
        public int NumberRole { get; set; }
        public bool IsAdmin { get; set; }
    }
}
